kratos/utilities/quaternion.h

```diff
@@ -117,49 +141,53 @@ namespace Kratos
                     mY = ae()(1);
                     mZ = ae()(2);
                     mW = ae()(3);
-                    
+
                     return *this;
                 }
 #endif // ifdef KRATOS_USE_AMATRIX
-		
+
 		///@}
-		
+
 	public:
 
 		///@name Access
 		///@{
-		
+
 		/**
 		Returns the X coefficient of this quaternion.
 		@return the X coefficient of this quaternion.
 		*/
 		inline const T x()const { return mX; }
-		
+		inline void Setx(T& value) { mX = value; }

```

Comment:
`SetX`?

I understand that the old x() was there, but maybe it is time to deprecate it.